corehq/apps/users/models.py

```diff
@@ -3282,6 +3282,14 @@ class ConnectIDUserLink(models.Model):
     connectid_username = models.TextField()
     commcare_user = models.ForeignKey(User, related_name='connectid_user', on_delete=models.CASCADE)
     domain = models.TextField()
+    messaging_consent = models.BooleanField(default=False)
```

Comment:
We should have a `channel_id` as well, since that is how incoming requests will be able to show which channel they go to.

Reply:
Added. https://github.com/dimagi/commcare-hq/pull/35212/commits/7d02e73e1f631a5182076b8f4ca41e4f8c585d68